frontend/rolecall/src/app/api/cast_api.service.ts

```diff
@@ -192,95 +199,73 @@ export class CastApi {
     return this.mockBackend.requestOneCast(uuid);
   };
 
-  /** Hits backend with create/edit cast POST request */
-  async requestCastSet(cast: Cast): Promise<HttpResponse<any>> {
-    if (environment.mockBackend) {
-      return this.mockBackend.requestCastSet(cast);
-    }
-    // Check if we have record of the cast and patch if we do
-    if (this.hasCast(cast.uuid)) {
-      // Do patch
-      let header = await this.headerUtil.generateHeader();
-      return this.http.delete(environment.backendURL + 'api/cast?castid=' + cast.uuid, {
+/** Hits backend with create/edit cast POST request */
+async requestCastSet(cast: Cast): Promise<HttpResponse<any>> {
+  if (environment.mockBackend) {
+    return this.mockBackend.requestCastSet(cast);
+  }
+  // Check if we have record of the cast and patch if we do
+  if (this.hasCast(cast.uuid)) {
+    // Do patch
+    let header = await this.headerUtil.generateHeader();
+    return this.http.delete(environment.backendURL + 'api/cast?castid=' + cast.uuid, {
+      headers: header,
+      observe: "response",
+      withCredentials: true
+    }).toPromise().catch((errorResp) => errorResp).then(
+        resp => this.respHandler.checkResponse<any>(resp)).then(async na => {
```

Comment:
Do you know why the response variable is called `na`? It looks like it's not used.

Reply:
I assume it means not used / not applicable. I changed it to notUsed.